book/website/reproducible-research/rdm/rdm-spreadsheets.md

```diff
@@ -89,7 +89,7 @@ Finally, you should also be aware of the default behaviour of your spreadsheet p
 For instance, the decimal is usually indicated with a comma in the French or German version of excel, but while a dot in English versions where the comma has no meaning
 (`9,000`  will be translated into 9000 or 9 depending on the version you are using).
 
```

Comment:
```suggestion
## 4. Data manipulation and analysis

***Do not manipulate or analyse data in a spreadsheet program.***

In particular, only copy-paste from one spreadsheet to another if the process is used very rarely. 
It is now very easy to read and combine the different spreadsheets in the analysis software, with the additional advantage that the software will return an error message if the headers do not fit.
```

Reply:
can't find how to delete the part 4 below

Reply:
You should be able to directly edit the file here: https://github.com/alan-turing-institute/the-turing-way/blob/malvikasharan-rdm-formatting/book/website/reproducible-research/rdm/rdm-spreadsheets.md

Reply:
this would make a new PR...

Reply:
It shouldn't. You will be able to directly edit in the branch that will appear in this PR.

Reply:
... made a new PR.

Reply:
merged to this 🌮